### PR TITLE
Add auto restore functionality to Rosetta unified image

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:
* Add auto restore functionality to Rosetta unified image
* Add documentation for running unified image
* Change Rosetta default start date to epoch
* Fix documentation bug with wrong units for `checkpoint_timeout`

**Related issue(s)**:

**Notes for reviewer**:
Need to test against a large db dump

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: docs/rosetta/README.md [hunk 1]
```diff
@@ -37,6 +37,116 @@ client.
...
+   file via `-e RESTORE=https://example.com/db.tar`. The database dump should be in the format specified in
+   the [backup](/docs/database.md#backup) section of the database upgrade documentation. The container database should
+   be empty otherwise the restore process will be skipped.
+
+5. Run the server from the all-in-one docker image with the appropriate `NETWORK` specified:
+
+```shell
+docker run -d -e MODE=online -e NETWORK=testnet \
+-v ./application.yml:/app/importer/application.yml \
+-p 5432:5432 -p 5700:5700 hedera-mirror-rosetta:0.45.0
```
Comment (Nana-EC) [inline docs/rosetta/README.md:92]: Did you wanna use the more recent version
```suggestion
-p 5432:5432 -p 5700:5700 hedera-mirror-rosetta:0.49.1
```

File: hedera-mirror-rosetta/build/run_supervisord.sh [hunk 2]
```diff
@@ -1,20 +1,67 @@
...
-  supervisord
+  echo "Running in online mode"
+  supervisord --configuration /app/supervisord.conf
+}
+
+function cleanup() {
+  /etc/init.d/postgresql stop || true
+  cp /app/postgresql.conf "${PGCONF}/conf.d"
+  rm -rf "${TMPDIR}"
+  cd /app
```
Comment (Nana-EC) [inline hedera-mirror-rosetta/build/run_supervisord.sh:18]: nit: Add debug echo to say successfully cleared temp files

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Looking good, minor clarification and suggestion

File: hedera-mirror-rosetta/build/run_supervisord.sh (hunk 2 above)
Comment (steven-sheehy) [reply hedera-mirror-rosetta/build/run_supervisord.sh:18]: Done

File: docs/rosetta/README.md (hunk 1 above)
Comment (steven-sheehy) [reply docs/rosetta/README.md]: Fixed

**Review by Nana-EC — APPROVED** [review]
LGTM

File: hedera-mirror-rosetta/build/run_supervisord.sh [hunk 3]
```diff
@@ -1,20 +1,68 @@
...
-  if [[ -n "$NETWORK" ]]; then
-    export HEDERA_MIRROR_IMPORTER_NETWORK=$NETWORK
-    export HEDERA_MIRROR_ROSETTA_NETWORK=$NETWORK
+  if [[ -n "${NETWORK}" ]]; then
+    export HEDERA_MIRROR_IMPORTER_NETWORK="${NETWORK}"
+    export HEDERA_MIRROR_ROSETTA_NETWORK="${NETWORK}"
   fi
 
-  case $MODE in
+  restore
```
Comment (xin-hedera) [inline hedera-mirror-rosetta/build/run_supervisord.sh:63]: only restore in online mode. offline mode just runs the rosetta server, without importer / db.

**Review by xin-hedera — CHANGES_REQUESTED** [review]

File: hedera-mirror-rosetta/build/run_supervisord.sh (hunk 3 above)
Comment (steven-sheehy) [reply hedera-mirror-rosetta/build/run_supervisord.sh]: Moved

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by Nana-EC — DISMISSED** [review]

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by Nana-EC — DISMISSED** [review]

**Review by Nana-EC — APPROVED** [review]

**Review by xin-hedera — APPROVED** [review]

**Pull request merged**